### PR TITLE
Add optional due date

### PR DESCRIPTION
**Pull request by hatappi**
I want to set due date, because my task often sets due date.

[Trello can set a due date.](http://help.trello.com/article/794-adding-due-dates-to-cards)
api also exists. [api reference](https://developers.trello.com/advanced-reference/card#put-1-cards-card-id-or-shortlink-due)
 
I implemented option that can set due date.

Please check PR 🙏

### CODE REVIEW

**Comment by gfx** [comment]
Can you add an example to README to suggest the valid date format?

File: README.md [hunk 1]
```diff
@@ -42,6 +42,17 @@ TRELLO_AUTO_ASSIGN - If set to '1', assigns sender with created card
...
+## Valid Value
+### dd \<due_date\>
+`YYYY-MM-DD` or `YYYY-MM-DD HH:MM`
+
+Example
+
+```
+dd 2016-01-01
+dd 2016-01-01 01:02
+```
```
Comment (hatappi) [inline README.md]: @gfx 
I added valid date format example to README.
Please check README..

Comment (gfx) [reply README.md]: Can you add a complete instruction to the **Usage** section?

e.g.

    ## Usage
    
    ```
    @ruboty trello b <board_name> l <list_name> (lb <label_name>) (dd <due_date>) c <card_name>
    ```

     e.g.

    ```
    @ruboty trello b development l icebox c something to do
    @ruboty trello b development l icebox dd 2016-01-01 01:02 c something to do
    ```

File: README.md [hunk 2]
```diff
@@ -21,9 +21,19 @@ Or install it yourself as:
...
 ```
 
+e.g.
+
+
+```
+@ruboty trello b development l icebox c something
+@ruboty trello b development l icebox dd 2016-01-01 c something
+@ruboty trello b development l icebox dd 2016-01-01 01:02 c something
+```
```
Comment (hatappi) [inline README.md:35]: @gfx 
Thank you for the example.
I wrote valid date example to usage.

**Comment by hatappi** [comment]
@gfx 
Sorry.
I did not add samples using all the options.
I added it.

**Comment by gfx** [comment]
Never mind. Thanks to the contribution. LGTM.

**Comment by dex1t** [comment]
LGTM 👍

**Comment by dex1t** [comment]
@hatappi released 💎 https://rubygems.org/gems/ruboty-trello/versions/0.1.6

**Pull request merged**